pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -169,8 +169,8 @@ if [ -n "$NIX_DEBUG" ]; then
   done
 fi
 
-if [ -n "$NIX_CC_WRAPPER_EXEC_HOOK" ]; then
-    source "$NIX_CC_WRAPPER_EXEC_HOOK"
+if [ -n "$NIX_@infixSalt@_CC_WRAPPER_EXEC_HOOK" ]; then
+    source "$NIX_@infixSalt@_CC_WRAPPER_EXEC_HOOK"
 fi
 
 PATH="$path_backup"
```

Comment:
@0xABAB I'm not sure what `NIX_@infixSalt@_CC_WRAPPER_EXEC_HOOK` is, but I don't know if it needs to be a store path. It could contain spaces.